tests/epyccel/test_epyccel_lists.py

```diff
@@ -117,3 +117,40 @@ def f():
 
     epyc_f = epyccel(f, language=language)
     assert f() == epyc_f()
+
+
+def test_clear_1(language):
+
+    def clear_1():
+        a = [1, 2, 3]
+        a.clear()
+        return a
+
+    epyc_last_element = epyccel(clear_1, language = language)
+    pyccel_result = epyc_last_element()
```

Comment:
Why do you use the name `epyc_last_element` for the pyccelized function? That does not make much sense to me. I suggest `epyc_clear_1`.

The same comment applies the following two test functions.

Reply:
fixed : b64b4e5